openwisp_monitoring/device/migrations/0002_create_template.py

```diff
@@ -0,0 +1,103 @@
+# Generated by Django 3.1.2 on 2020-12-20 06:15
+
+import uuid
+from collections import OrderedDict
+
+from django.db import migrations
+from django.db.models import Q
+
+
+def migrate_data(apps, schema_editor):
+    Template = apps.get_model('config', 'Template')
+    if Template.objects.filter(
+        Q(config__contains='/usr/sbin/openwisp-monitoring')
+        & Q(config__contains='/usr/sbin/netjson-monitoring'),
+        default=True,
+    ).exists():
+        return
+    default_template = Template(
+        # The UUID has been fixed so that the template could be auto-updated
+        pk=uuid.UUID('66b1a809-2d83-4e33-8b2d-64aa5d4036a9'),
```

Comment:
store this UUID in `device/migrations/__init__.py` so it can be called from future migrations.

Let's make it also obviously artificial, so it will be easily recognizable by us when debugging, eg: `uuid.UUID('00000000-defa-defa-defa-000000000000')`